systemtest/src/test/java/io/strimzi/systemtest/kafka/listeners/ListenersST.java

```diff
@@ -68,14 +68,15 @@
 public class ListenersST extends AbstractST {
     private static final Logger LOGGER = LogManager.getLogger(ListenersST.class);
 
-    public static final String NAMESPACE = "kafka-listeners-cluster-test";
+    public static final String NAMESPACE = "listeners";
```

Comment:
Is this intention? Just asking..

Reply:
Yes, the previous name was problematic, because route name was too long.

Reply:
Yep okay, thanks.